docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -0,0 +1,109 @@
+.. role:: underline
+    :class: underline
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+  
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+.. _bliss:
+
+==================
+BLISS Style Guide
+==================
+
+Building Language Inclusivity into Simulation Science (BLISS) is an important part of
+our team's work! Here you will find our ongoing style guide for incorporating inclusive 
+language into our day-to-day work.
+
+
+1.0 Overview 
+++++++++++++
+
+1.1 Guiding Principles
+----------------------
+In this section, we will outline the guiding principles of BLISS.
+
+1.2 Glossary of General Terminology
+-----------------------------------
+In this section, we will create a glossary of general terminology that will be used 
+throughout this style guide (e.g., for starters, we should explicitly define sex and 
+gender; SGM).
+
+1.3 Visuals
```

Comment:
Personally, I'd prefer to have visuals (which will mostly be tables, right?) sprinkled throughout instead of in their own section.

For the examples listed below, I think the dos and don'ts could probably fit into Guiding Principles. And the "what this is and what this isn't" feels like it could fit into an introductory paragraph at the start of Overview.

Reply:
I agree!